packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx

```diff
@@ -57,11 +62,7 @@ import { SvgSplit } from '../../../icons/v0';
 import { SvgAdd, SvgPiggyBank, SvgTrash } from '../../../icons/v1';
 import { SvgPencilWriteAlternate } from '../../../icons/v2';
 import { useSelector, useDispatch } from '../../../redux';
-import { styles, theme } from '../../../style';
-import { Button } from '../../common/Button';
```

Comment:
This is what breaks the VRT tests. `common/Button` !== `common/Button2`

Reply:
I see, I just saw this comment (in `Button.tsx`) and thought it was the same
![image](https://github.com/user-attachments/assets/b459015e-f4fa-48e5-8e8b-af31b0d3ac6a)

Reply:
I can see that now the test passed but https://deploy-preview-4385.demo.actualbudget.org/transactions/new still shows a blank page on mobile